fe/fe-core/src/main/java/org/apache/doris/alter/AlterLSCHelper.java

```diff
@@ -79,7 +80,16 @@ public void enableLightSchemaChange() throws DdlException {
         final Map<Long, PFetchColIdsRequest> params = initParams();
         final PFetchColIdsResponse response = callForColumnIds(params);
         updateTableMeta(response);
```

Comment:
1、make `updateTableMeta(response)`  like this updateTableMeta(AlterLightSchemaChangeInfo info) so that it can be reused in replay